lib/configuration/applyConfiguration.js

```diff
@@ -0,0 +1,111 @@
+const R = require('ramda');
+const { EventEmitter } = require('events');
+const logger = require('../logger');
+const getProxySettings = require('../getProxySettings');
+const applyLoggingOptions = require('./applyLoggingOptions');
+const validateConfig = require('./validateConfig');
+const { normalizeConfig } = require('./normalizeConfig');
+
+const defaultConfig = {
```

Comment:
I wonder if this shouldn't be called `DEFAULT_CONFIG` so it's clearer in the code below this is a global static thing rather than a local variable

Reply:
No hard opinion on that :) We are not reusing it anywhere, so it's not global per say, it remains scoped to this module.

Reply:
We're exporting it and using it in the respective test file and I think the constant-like letter casing would convey better what the variable represents